spec/services/trainees/create_from_dttp_spec.rb

```diff
@@ -516,23 +516,33 @@ module Trainees
       context "when trainee already exists" do
         before { create(:trainee, dttp_id: dttp_trainee.dttp_id) }
 
+        it "application is not imported" do
+          expect {
+            create_trainee_from_dttp
+          }.not_to change(Trainee, :count)
+        end
+
```

Comment:
Are these additional specs just along for the ride? :P Not that they're bad, but do they belong in another commit?